nginx/default.dev.conf

```diff
@@ -11,7 +11,7 @@ server {
         proxy_pass http://backend:4000;
         proxy_redirect off;
 
-        proxy_cookie_path / "/; secure; HttpOnly; SameSite=strict";
+        proxy_cookie_path / "/; HttpOnly; SameSite=strict";
```

Comment:
removed because setting cookies to `secure` in development will cause them not to be sent to Infisical (since connection is via HTTP)

Reply:
based on the nginx documentation here https://nginx.org/en/docs/http/ngx_http_proxy_module.html

proxy_cookie_path only rewrites the path and nothing more.